src/test/java/qna/domain/AnswerTest.java

```diff
@@ -1,6 +1,55 @@
 package qna.domain;
 
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DataJpaTest
 public class AnswerTest {
     public static final Answer A1 = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
     public static final Answer A2 = new Answer(UserTest.SANJIGI, QuestionTest.Q1, "Answers Contents2");
+
+    @Autowired
+    AnswerRepository answers;
```

Comment:
중요한 부분은 아니지만, 접근 제한자가 빠졌네요.

```suggestion
    private AnswerRepository answers;
```